Projects/Server/Buffers/SpanReader.cs

```diff
@@ -165,12 +165,19 @@ public ulong ReadUInt64()
         return value;
     }
 
+    /// <summary>
+    /// Check the string in the buffer without converting it with the provided encoding.
+    /// </summary>
+    /// <param name="encoding">Text encoding used for char size calculation</param>
+    /// <param name="fixedLength">Max buffer length to check</param>
+    /// <returns>Byte length of the found string</returns>
+    /// <exception cref="OutOfMemoryException"></exception>
     [MethodImpl(MethodImplOptions.AggressiveInlining)]
-    public string ReadString(Encoding encoding, bool safeString = false, int fixedLength = -1)
+    public int SkipString(Encoding encoding, int fixedLength = -1)
```

Comment:
Per our conversation, this should be an internal implementation detail. I am ok with a private method called something like `GetStringLength()`, and we _don't_ update the `Position`, then we can do an actual ReadBytes() -> to string for that length.

For public consumption, let's just use `Seek`, especially since this is UTF-16 and we can assume the byte length.